### DISCUSSION TITLE
3.3.3

### DISCUSSION THREAD

**Discussion by mathiascode** [nicotine-plus/nicotine-plus#2999] [Releases]
WINDOWS USERS: The installer format has changed in Nicotine+ 3.2.0. If you are upgrading from Nicotine+ 3.1.1 or earlier, please uninstall Nicotine+ first (this will not remove your existing settings).

### Changes

 * Performance improvements when searching for common files
 * Improved search result matching for non-Latin languages
 * Minor accessibility improvements
 * Updated translations
 * GTK 3: Restored X11 tray icon implementation for compatibility with older systems

### Corrections

 * Fixed a rare crash when changing grouping mode in transfer tabs
 * Fixed a rare crash when closing Browse Shares tab
 * Fixed a rare crash when initializing an upload
 * Fixed a rare crash when double-clicking a download
 * Fixed an issue where passwords were not remembered after changing them
 * Fixed an issue where number of shared files was only updated on startup (regression in 3.3.2)
 * Fixed an issue where transfers displayed incorrect "User logged off" status in some cases
 * Fixed an issue where chat history entry completion no longer worked
 * Fixed an issue where UPnP did not work on MikroTik routers
 * Fixed missing button labels in certain dialogs
 * Fixed some small memory leaks
 * Avoid selecting new transfers while tab is active
 * Remember selected folder when refreshing local shares
 * Windows: Fixed Alt+1-9 tab shortcuts interfering with Alt codes
 * Windows: Fixed window not minimizing when clicking task bar icon
 * Windows: Fixed title bar buttons requiring two clicks to activate
 * Snap: Fixed an issue where the file chooser displayed the wrong initial folder
 * Snap: Fixed an issue where folders shared from external drives were unavailable after remounting
 * Termux: Fixed an issue where connecting to the server failed

### Issues closed on GitHub

 * Bad user experience with Windows-like buttons on Nicotine 3.3.0 ([#2880](https://github.com/nicotine-plus/nicotine-plus/issues/2880))
 * Auto-selection of new transfers ([#2901](https://github.com/nicotine-plus/nicotine-plus/issues/2901))
 * Nicotine+ 3.3.2 displays user as logged off while the transfers are active ([#2909](https://github.com/nicotine-plus/nicotine-plus/issues/2909))
 * No icon in taskbar when running in background ([#2928](https://github.com/nicotine-plus/nicotine-plus/issues/2928))
 * Set 'Finished' status instead of 'Filtered' for finished transfers in case the directory had files matching download filtering pattern ([#2932](https://github.com/nicotine-plus/nicotine-plus/issues/2932))
 * Refresh Files button not working ([#2937](https://github.com/nicotine-plus/nicotine-plus/issues/2937))
 * Nictotine Hard fail trying to get pointer ([#2943](https://github.com/nicotine-plus/nicotine-plus/issues/2943))
 * 3.3.2 - System Tray Icons Don't Appear ([#2944](https://github.com/nicotine-plus/nicotine-plus/issues/2944))
 * Sudden crash ([#2950](https://github.com/nicotine-plus/nicotine-plus/issues/2950))
 * bug on checking transfers ([#2953](https://github.com/nicotine-plus/nicotine-plus/issues/2953))
 * UPnP: Failed to forward external port 2234: HTTP Error 500: Internal Server Error ([#2955](https://github.com/nicotine-plus/nicotine-plus/issues/2955))
 * 0 Shares on Profile ([#2956](https://github.com/nicotine-plus/nicotine-plus/issues/2956))
 * Crash when double clicking download row ([#2961](https://github.com/nicotine-plus/nicotine-plus/issues/2961))
 * Nicotine+ 3.3.2 permission denied to port 2234 on Fedora 39 ([#2968](https://github.com/nicotine-plus/nicotine-plus/issues/2968))
 * This option is doesn't work on Nicotine+ 3.3.3 rc2 ([#2969](https://github.com/nicotine-plus/nicotine-plus/issues/2969))
 * Num lock state is inverted ([#2977](https://github.com/nicotine-plus/nicotine-plus/issues/2977))
 * Random crash ([#2982](https://github.com/nicotine-plus/nicotine-plus/issues/2982))
 * Found another bug ([#2988](https://github.com/nicotine-plus/nicotine-plus/issues/2988))
 * Arbitrary Critical Error ([#2996](https://github.com/nicotine-plus/nicotine-plus/issues/2996))

<hr /><em>This discussion was created from the release <a href='https://github.com/nicotine-plus/nicotine-plus/releases/tag/3.3.3'>3.3.3</a>.</em>

**Comment by majkinetor** [comment]
FYI, had to revert to previous version because its so slow its unusable - any action has lag of around 1s - scrolling, clicking an item in the list, switching tabs etc (Windows 10 OS, portable version)

**Reply by mathiascode** [reply]
Yeah, looks like it, and of course it made it into a release. I give up.

**Reply by mathiascode** [reply]
Please try 3.3.4rc1 and see if it's fixed on your end.
https://nicotine-plus.org/doc/TESTING.html#windows

**Reply by majkinetor** [reply]
That seems to work. Thanks @mathiascode
